zcash_primitives/src/consensus.rs

```diff
@@ -659,9 +659,13 @@ mod tests {
             BranchId::for_height(&MAIN_NETWORK, BlockHeight(1_046_400)),
             BranchId::Canopy,
         );
+        assert_eq!(
+            BranchId::for_height(&MAIN_NETWORK, BlockHeight(1_687_104)),
+            BranchId::Nu5,
+        );
         assert_eq!(
             BranchId::for_height(&MAIN_NETWORK, BlockHeight(5_000_000)),
-            BranchId::Canopy,
+            BranchId::Nu5,
```

Comment:
Should this be "NU5"?

Reply:
Nm, looks like the constant is mixed case.

Reply:
Yes; Rust convention is to use CamelCase even for acronyms.